src/analysis/score.rs

```diff
@@ -24,6 +30,94 @@ pub fn row_accuracy(outputs: &Matrix<f64>, targets: &Matrix<f64>) -> f64 {
     accuracy(outputs.iter_rows(), targets.iter_rows())
 }
 
+/// Returns the precision score for 2 class classification.
+/// true-positive / (true-positive + false-positive)
```

Comment:
We should add a [`# Panics` section](https://doc.rust-lang.org/book/documentation.html#special-sections) explaining how the function can panic.